test/functional/feature_csv_activation.py

```diff
@@ -90,7 +90,6 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.extra_args = [[
             '-whitelist=noban@127.0.0.1',
-            '-acceptnonstdtxn=1',
```

Comment:
Nice. Thanks for working on this.

I guess an alternative to consider might be to use the option to ignore specific reject reasons, when it is availalbe. #20753